### PR TITLE
PIKIP-10 Styling and small fixes

### PR DESCRIPTION
**Pull request by rokaskasperavicius**

### CODE REVIEW

File: app/CreateForm.jsx [hunk 1]
```diff
@@ -17,55 +16,78 @@ import {
...
 
 class CreateForm extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      response: '',
+      status: undefined,
+      response: undefined,
     };
```
Comment (ChristianMeiselsAsmussen) [inline app/CreateForm.jsx]: Why initiate as undefined? Why not just  not define them?

**Review by ChristianMeiselsAsmussen — CHANGES_REQUESTED** [review]
Nothing serious - just a small comment

File: app/CreateForm.jsx (hunk 1 above)
Comment (rokaskasperavicius) [reply app/CreateForm.jsx]: The status and the response come from the API call and the status can be either "success" or "error", so I don't really know how to initiate without using "undefined" or "null".

Comment (rokaskasperavicius) [reply app/CreateForm.jsx]: And the response can be either an error text or on success generated profile link

**Pull request merged**